app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,102 +1,86 @@
+// @ts-nocheck
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
+import Item from "components/Common/Item";
```

Comment:
Since it's a `Common` folder there will be more than one component that will be exported. Thus create an `index.js` and export all items from that folder using it.

Reply:
> Thus create an `index.js` and export all items from that folder using it

Where are you suggesting to create an `index.js`?
I didn't understand the whole thing. Would you mind sharing the reference to a repo, which has incorporated this practice.

Reply:
Thanks for sharing these resources.
Understood and implemented.

Q: Do you have all these blogs/article ready beforehand or prefer to google on the go?

Reply:
@sachinjoshi12 I google on the go. 😄